### ISSUE TITLE
Display different copy on Ongoing Recruitment advertisement page

### ISSUE THREAD

**Issue by tristan-orourke** [GCTC-NTGC/gc-digital-talent#4933]
## Job poster context

Because ongoing recruitment will have slight administrative differences for candidates, we need to alter the context accordion at the top of the poster to provide comparable information about how ongoing recruitment works.

Copy for this is still in the works: nudge @substrae when this becomes critical

![image](https://user-images.githubusercontent.com/6960386/204003292-41ee125d-a80b-4084-b411-660605c596a9.png)

## Acceptance Criteria
- [ ] Show different copy in context accordion if job belongs to the IT_JOBS_ONGOING publishing group.

**Comment by mnigh** [comment]
> Copy for this is still in the works: nudge @substrae when this becomes critical

Is this issue blocked by another issue? I just checked and it is currently possible to publish a pool with a value of `IT_JOBS_ONGOING` in the publishing_group field.

**Comment by substrae** [comment]
I think we decided to just leave the same copy on both... @gobyrne?

**Comment by mnigh** [comment]
> I think we decided to just leave the same copy on both... @gobyrne?

@gobyrne can you confirm? if so, this issue can be closed as stale.

**Comment by gobyrne** [comment]
Yeah, the copy actually works better for ongoing recruitment than the original process. We'll open another issue if needed.
